optimum/neuron/modeling_diffusion.py

```diff
@@ -641,6 +641,12 @@ def _from_pretrained(
                 }
             )
             # Downloads all repo's files matching the allowed patterns
+            is_flux_model = "flux" in model_id.lower()
+            if is_flux_model:
+                ignore_patterns = ["*.msgpack", "*.bin"]
+            else:
+                ignore_patterns = ["*.msgpack", "*.safetensors", "*.bin"]
+
```

Comment:
Maybe it's not necessary, let's just set the `ignore_patterns = ["*.msgpack", "*.bin"]`

Reply:
I think it should be applicable for all compiled neuron models for diffusers. sd and pixart won't have safetensors anyway